styles.css

```diff
@@ -1393,3 +1344,109 @@ textarea {
     color: #b3bac5;
   }
 }
+
+.container-popup1 {
+  position: relative;
+}
+
+.popup-row {
+  position: fixed;
+  top: 1rem;
+  right: 1rem;
+  left: 1rem;
+  bottom: 1rem;
+  display: flex;
+  flex-direction: column;
+  padding: 16px;
+  gap: 12px;
+  background: #fff;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+  max-height: 100%;
+  z-index: 11;
+  overflow-y: scroll;
+}
+
+.overlay {
+  display: none;
+  position: absolute;
+  top: 0;
+  left: 0;
+  height: 100%;
+  width: 100%;
+  z-index: 10;
+  background-color: rgba(0, 0, 0, 0.6);
+}
+
+h3.popup-header {
+  margin-right: auto;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  color: #172b4d;
+}
+
+.action-buttons {
+  display: flex;
+  justify-content: space-evenly;
+}
+
+.seelive {
+  margin-right: 2rem;
+}
+
+.close-popup img {
+  position: absolute;
+  top: 2.7rem;
+  right: 2rem;
+}
+
+.try-me {
+  width: 100%;
+  height: 100vh;
+  position: absolute;
+
+  /* background-color: black; */
+  top: 0;
+  left: 0;
+}
+
+.try-me.active {
+  display: flex;
+}
+
+.big-btn {
+  height: 48px;
+  width: 146px;
+  border-radius: 10px;
+  border: 2px solid #6070ff;
+  background-color: #fff;
+  color: #396df2;
+  font-family: "poppins", sans-serif;
+  font-weight: 500;
+  font-size: 15px;
+}
+
+.live {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  bottom: 50%;
+}
+
+.close-popup {
+  width: 20px;
+  color: #000;
+  font-size: 20px;
+}
+
+.popup-header {
+  display: flex;
+  justify-content: space-between;
+  width: 100%;
+  padding: 5px 20px;
+}
+
+.none {
+  display: none;
+}
```

Comment:
- [ ] Please consider re-arranging the elements in the pop-up for mobile to match the Figma design as close as possible:
- please consider styling the  buttons as per the Figma design - please keep in mind to implement different styles for user interactions;
- kindly, don't forget to style the technologies as they are styled in the Figma design;
- also, please place the buttons and the technologies in the right place according to the Figma design;

**Your design:**
![image](https://user-images.githubusercontent.com/83596135/180838887-5550a567-829c-4795-a7dd-77abda6429e8.png)


**Figma design:**
![image](https://user-images.githubusercontent.com/83596135/180838977-dc84325f-76a1-4829-b8ee-043313e6cbff.png)